dashboard/src/components/ServiceInstanceView/ServiceInstanceView.tsx

```diff
@@ -198,45 +193,67 @@ class ServiceInstanceView extends React.Component<IServiceInstanceViewProps> {
     }
 
     return (
-      <div className="container">
-        <PageHeader>
-          <h1>{name}</h1>
-        </PageHeader>
+      <section className="ServiceInstanceView padding-b-big">
         <main>
           <LoadingWrapper loaded={loaded}>
-            {this.props.errors.fetch && (
-              <ErrorSelector
-                error={this.props.errors.fetch}
-                resource={`Instance ${name}`}
-                action="list"
-                defaultRequiredRBACRoles={RequiredRBACRoles}
-              />
-            )}
-            {this.props.errors.deprovision && (
-              <ErrorSelector
-                error={this.props.errors.deprovision}
-                resource={`Instance ${name}`}
-                action="deprovision"
-                defaultRequiredRBACRoles={RequiredRBACRoles}
-              />
-            )}
-            {instance && (
-              <div className="found">
-                <h2>About</h2>
-                <div>{body}</div>
-                <DeprovisionButton deprovision={deprovision} instance={instance} />
-                <h3>Spec</h3>
-                <CardGrid>
-                  {classCard}
-                  {planCard}
-                </CardGrid>
-                <h2>Bindings</h2>
-                {bindingSection}
-              </div>
-            )}
+            <div className="container">
+              {this.props.errors.fetch && (
+                <ErrorSelector
+                  error={this.props.errors.fetch}
+                  resource={`Instance ${name}`}
+                  action="list"
+                  defaultRequiredRBACRoles={RequiredRBACRoles}
+                />
+              )}
+              {this.props.errors.deprovision && (
+                <ErrorSelector
+                  error={this.props.errors.deprovision}
+                  resource={`Instance ${name}`}
+                  action="deprovision"
+                  defaultRequiredRBACRoles={RequiredRBACRoles}
+                />
+              )}
+              {instance && (
+                <div className="row collapse-b-tablet">
+                  <div className="col-12">
+                    <MessageAlert level="warning">
+                      <div>
+                        <div>Refresh the page to update the status of this Service Instance.</div>
```

Comment:
Maybe this message is too visible? Is kind of common that if you don't see thing being updated you can just refresh.

Reply:
The reason I added it is because now we have the flashing icon when it's provisioning or deprovisioning, which could mislead people to think that it will auto-update. They also might expect it to act in the same way as the AppView page. I think it's good to have there (though we can remove the line about Service Catalog integration being experimental if you think the message should be shorter). Another option is to remove the flashing provisioning/deprovisioning icon and just make it static. WDYT?

Reply:
yes, maybe a more subtle thing is to remove the "flashing" behavior. In any case, is that is somehow difficult it's okay to leave it as it is.

Reply:
I think we can keep this for now, and we can revisit it based on feedback from @arapulido and others.